src/layers/EarthEngine.js

```diff
@@ -415,7 +415,7 @@ class EarthEngine extends Layer {
         const layerId = `${id}-polygon`
         const mapgl = this.getMapGL()
 
-        if (mapgl.getLayer(layerId)) {
+        if (mapgl && mapgl.getLayer(layerId)) {
```

Comment:
```suggestion
        if (mapgl?.getLayer(layerId)) {
```

Reply:
I agree, but I don't take the risk on hard freeze day - need do be sure that it's supported by the babel setup first.